### ISSUE TITLE
removing gcm provider require the apikey

### ISSUE THREAD

**Issue by linuxjuggler** [uniqush/uniqush-www#1]
hey,
the documentation at the website and [here](http://d.pr/119gh) says that for removing gcm you dont need the  **apikey**, but when executing **without** the apikey I got this message

````
[RemovePushServiceProvider][Error] 2014/12/02 08:56:16 From=xxx.xxx.xxx.xxx:xxxx Cannot build push service provider: NoAPIKey
````

and I have provided the following:
```
curl -X "POST" "http://xxx.xxx.xxx.xxx:xxxx/rmpsp" \
	-d "service=testing" \
	-d "pushservicetype=gcm" \
	-d "projectid=1043028787"
```

so am not sure is it a bug in the program or in the documentations.
thanks

**Comment by mishan** [comment]
@linuxjuggler We've just released 2.1.0 -- see releases on GitHub (I still need to update the blog). If you're still having this problem, could you try with the new version? If it still happens, it should at least have a much better description of the error. Sorry, didn't notice your issue since it's filed on the `uniqush-www` repo instead of `uniqush-push`

Please open an issue on `uniqush-push` if this is still happening with output from the latest version

**Comment by linuxjuggler** [comment]
@mishan no problem I will see and let you know soon, I figured out its maybe an error in the docs thats why I opened it here .
